src/main/java/com/plivo/api/models/base/ListResponse.java

```diff
@@ -19,10 +20,14 @@ public Meta getMeta() {
     return meta;
   }
 
+  public void updateMeta(MessageMeta meta) {
```

Comment:
These setter method should not exist and even if they do it should not be public. Is there a way to instantiate the object without these?